bin/clu

```diff
@@ -10,12 +10,13 @@ if ( file_exists( $path = __DIR__ . '/../vendor/autoload.php' )
 	throw new \Exception('Could not locate autoload.php');
 }
 
-\CLU\Checker::check_executables();
-
 // Simple argv parsing
 $optind = null;
-$opts = getopt('', ['security-only'], $optind);
+$opts = getopt('', ['security-only', 'gitlab'], $optind);
```

Comment:
Instead of a `gitlab` option let's use a `git_provider` option that defaults to `github`.